### ISSUE TITLE
Whether it is possible to do semantic segmentation without prompts or dots

### ISSUE THREAD

**Issue by lucky-asia** [facebookresearch/segment-anything#793]
Can you do not need a hint when inferring, that is, directly input the image to predict the segmentation result, and do not need to pass in coordinate information. That is, no hints are required, just like a normal semantic segmentation model

**Comment by ywangwxd** [comment]
Yes. You can check documentation about SamAutomaticMaskGenerator.
It will generate segmentation mask without any bbox or point as input.

The underlying mechanism is to predefine some grid based positive points from the image and use these points as point input.
